frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -221,4 +227,25 @@ describe('Kubevirt create VM using wizard', () => {
     },
     CLONE_VM_TIMEOUT_SECS,
   );
+
```

Comment:
can we include this test in the CI (kubevirt-gating)?

Reply:
should I copy the test elsewhere?

Reply:
you can try to add it to integrationTestSuites in our package.json. 

If for some reason the other tests begin to fail - we can skip them and merge just yours only and focus on the skipped tests later.